chapters/fragops.txt

```diff
@@ -955,22 +956,24 @@ fragments generated by <<primsrast-polygons-basic,back-facing>>
 fragments generated by <<primsrast-polygons-basic,front-facing polygons>> or
 any other primitives.
 
-The comparison performed is based on the elink:VkCompareOp, compare mask
-[eq]#s~c~# , and stencil reference value [eq]#s~r~# of the relevant state
-set.
-The compare mask and stencil reference value are set by either the
+The comparison operation performed is determined by a elink:VkCompareOp
+value set by
+ifdef::VK_VERSION_1_3,VK_EXT_extended_dynamic_state[]
+flink:vkCmdSetDepthCompareOp::pname:depthCompareOp, or by
```

Comment:
The flink: macro will rewrite vkCmdSetStencilOp -> vkCmdSetStencilOpEXT if the former is not supported at build time but the latter is - see LinkInlineMacroBase in config/spec-macros/extension.rb - so that's what comes out in the 1.0-extensions build. That contributed to greatly reducing the number of dead links in the older spec builds when it was introduced.